liquidctl/driver/kraken3.py

```diff
@@ -885,6 +923,35 @@ def prepare_frames(frames):
 
         return result_bytes.getvalue()
 
+    def _send_data_fw2(self, data, bulkInfo):
```

Comment:
I'd add an explicit comment that this is for firmware 2.X, and rename to `_send_elite_data_fw2()` or similar. (Yes, it's obvious from the name now, but still.)